app/src/main/java/foi/air/szokpt/views/app/TransactionDetailsView.kt

```diff
@@ -0,0 +1,253 @@
+package foi.air.szokpt.views.app
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.Info
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.navigation.NavController
+import foi.air.szokpt.R
+import foi.air.szokpt.ui.components.TileSegment
+import foi.air.szokpt.ui.theme.BGLevelOne
+import foi.air.szokpt.ui.theme.Primary
+import foi.air.szokpt.ui.theme.TextWhite
+import foi.air.szokpt.ui.theme.TileSizeMode
+import hr.foi.air.szokpt.ws.models.responses.Transaction
+
+@Composable
+fun TransactionDetailsView(
+    navController: NavController,
+    transaction: Transaction
+) {
+    val currencySymbol = when (transaction.currency) {
+        "840" -> "$"
+        "978" -> "€"
+        else -> ""
+    }
+
+    val trxTypeMap = mapOf(
+        "sale" to "Sale",
+        "refund" to "Refund",
+        "void_sale" to "Void sale",
+        "void_refund" to "Void refund",
+        "reversal_sale" to "Reversal sale",
+        "reversal_refund" to "Reversal refund"
+    )
+
+    val cardBrandDrawable = when (transaction.cardBrand) {
+        "Maestro" -> R.drawable.maestro
+        "Visa" -> R.drawable.visa
+        "MasterCard" -> R.drawable.mastercard
+        "Diners" -> R.drawable.diners
+        "Discover" -> R.drawable.discover
+        else -> R.drawable.logo
+    }
```

Comment:
Bilo bi dobro napraviti klasu (npr. TransactionUtils) koja bi sadržavala metode getCurrencySymbol, getTransactionType, getCardBrandDrawable i getCurrencyColor. Razlog tome jest što se ovaj kod već koristi u TransactionItem komponenti pa bi bilo dobro izdvojiti ga da se ne ponavlja.